Makefile

```diff
@@ -78,6 +78,13 @@ DESTTARGET := $(abspath $(DESTBINDIR)/$(TARGET))
 DESTCONFDIR := $(DESTDIR)/$(SYSCONFDIR)/$(CCDIR)
 DESTCONFIG := $(abspath $(DESTCONFDIR)/$(CONFIG_FILE))
 
+PAUSEDIR := pause
+PAUSEBIN := pause
```

Comment:
I find these two variables rather confusing to be honest. Do we really need them?